### PR TITLE
Add test to profile page

### PR DESCRIPTION
**Pull request by vithura14**
Result of tests are in a video in our Google drive. If you wish it to run it by yourself, you can run:
`allure open`

### CODE REVIEW

File: src/screens/Messages/Tabs/TeamChat.js [hunk 1]
```diff
@@ -74,16 +74,16 @@ class TeamChat extends Component {
 
     removeTeam = async (room) => {
 
-        let token = await AsyncStorage.getItem("token")
-        let user = await AsyncStorage.getItem("userID")
+        let userID = "eaf54fae-1ab8-4b5a-8047-51904f6ae884"
```
Comment (car12lin12) [inline src/screens/Messages/Tabs/TeamChat.js]: do these need to be changed for the reports to be generated? It shouldn't be changed for the actual app, but only for testing

File: App.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ import MessagesIconFill from './src/assets/images/icons/message_bubble_fill.png'
 import FeedIconFill from './src/assets/images/icons/feed_fill.png';
 import ScheduleIconFill from './src/assets/images/icons/calendar_fill.png';
 import ProfileIconFill from './src/assets/images/icons/user_fill.png';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```
Comment (car12lin12) [inline App.js]: please remove all changes to App.js for regular app usage

**Review by car12lin12 — CHANGES_REQUESTED** [review]
Great job Vithu :smile: I noticed you committed changes from App.js and TeamChat.js, even though used for testing, the changes cannot be kept for regular app usage. Can we remove the changes and still keep the auto generated reports?

**Comment by vithura14** [comment]
> Great job Vithu 😄 I noticed you committed changes from App.js and TeamChat.js, even though used for testing, the changes cannot be kept for regular app usage. Can we remove the changes and still keep the auto generated reports?

Yes definitely! Will address your PR comments right away!

**Review by car12lin12 — APPROVED** [review]

**Pull request merged**